src/config.ts

```diff
@@ -182,6 +183,67 @@ export const defaultTranslators: TranslatorConfigObject = {
     }
   },
 
+  /* Table */
+  'table': ({}) => {
+    const tableContentTranslators = new TranslatorCollection();
```

Comment:
For speed purposes, we should probably affix `tableContentTranslators` to `NodeHtmlMarkdown` like the other translator collections, so it does not have to be recreated each time. But, let me know if I'm missing something. Was there a reason that you chose to put it here?

Reply:
Indeed. It was useless to create every single instance.

I thought it would be better to narrow the scope because it is unlikely to be used in other codes, but as you say, it is better to declare it out there.